ansible.cfg

```diff
@@ -3,6 +3,6 @@ host_key_checking = no
 record_host_keys = no
 ssh_args = -o ControlMaster=auto -o ControlPersist=30m
 callback_plugins = plugins/callbacks
-hostfile = inventory/1-datacenter
+hostfile = plugins/inventory
```

Comment:
I can see it both ways. It's probably fine how it is, for now... if we get people confused about it we'll just change.